portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/questions/QuestionsWidgetActions.testcase

```diff
@@ -402,6 +402,175 @@ definition {
 		}
 	}
 
+	@description = "Validate if the user can subscribe and unsubscribe from the question."
+	@priority = "5"
+	test SubscribeAndUnsubscribeFromQuestion {
+		property custom.properties = "jsonws.web.service.paths.excludes=";
+
+		JSONUser.addUser(
+			userEmailAddress = "userea@liferay.com",
+			userFirstName = "userfn",
+			userLastName = "userln",
+			userScreenName = "usersn");
+
+		JSONUser.setFirstPassword(
+			agreeToTermsAndAnswerReminderQuery = "true",
+			requireReset = "false",
+			userEmailAddress = "userea@liferay.com");
+
+		User.openUsersAdmin();
+
+		User.assignRegularRoleCP(
+			roleTitle = "Administrator",
+			userScreenName = "usersn");
```

Comment:
Something we can do here in the future is to refactor this so it's assigned through the backend. Ideally we are not doing any test setup in the UI.